src/main/java/com/zaxxer/nuprocess/windows/ProcessCompletions.java

```diff
@@ -69,6 +71,25 @@
 
    ProcessCompletions()
    {
+      this(LINGER_ITERATIONS);
+   }
+
+   ProcessCompletions(WindowsProcess process)
+   {
+      this(-1);
+
+      if (process == null) {
+         throw new IllegalArgumentException("process");
+      }
+
+      registerProcess(process);
+      checkAndSetRunning();
+   }
+
+   private ProcessCompletions(int lingerIterations)
```

Comment:
This constructor already existed on `BaseEventProcessor`, and the `ProcessEpoll` and `ProcessKqueue` subclasses, but we had to add it on `ProcessCompletions`. Without the ability to specify the `lingerIterations`, the current hard-coded value applies a 2.5 _second_ penalty for every synchronous process. (Unless you override it with a system property, of course, but at that point you're also reducing iterations for async processes.)